audit-ci.json

```diff
@@ -12,6 +12,8 @@
     "knex",
     "jsonwebtoken",
     "cacheable-request",
-    "http-cache-semantics"
+    "http-cache-semantics",
+    "lodash.pick",
+    "crypto-js"
```

Comment:
Looks like @Jkovarik also has these changes so it may just be whomever gets there first: https://github.com/nasa/cumulus/pull/3565/files#diff-3067268e04946afc99fcc6d204f22cc7105a282f106c6eda212a85d8dd3e5ba2R17